docs/protocol/concepts/tokens.md

```diff
@@ -12,14 +15,27 @@ This matches Ethereum denomination of:
 
 1 ETH = 10<sup>18</sup> wei
 
-
 ### Cosmos Coins
 
-Accounts can own Cosmos coins in their balance, which are used for operations with other Cosmos and transactions. Examples of these are using the coins for staking, IBC transfers, governance deposits and EVM.
+Accounts can own Cosmos coins in their balance, which are used for operations with other Cosmos and transactions. Examples
+of these are using the coins for staking, IBC transfers, governance deposits and EVM.
 
 ### EVM Tokens
 
 Evmos is compatible with ERC20 tokens and other non-fungible token standards (EIP721, EIP1155)
 that are natively supported by the EVM.
 
+## Evmos Dashboard
+
+Check out how we represent ERC-20 tokens and Cosmos IBC Coins through our [Single Token Representation](https://app.evmos.org/assets)
+feature on the Evmos Dashboard.
```

Comment:
Would elaborate that on Evmos we want to enable a UX that doesn't require users to know about the underlying asset type. Instead, users only see the denomination and amount of that asset. The protocol handles conversions in the background. 

A link to token registration would be helpful too: https://main--dancing-hummingbird-242a98.netlify.app/develop/mainnet#token-registration